seednode/torrc

```diff
@@ -1,18 +1,144 @@
-RunAsDaemon 1
```

Comment:
Should this be present in the new tor config?

Reply:
This is done by `/usr/share/tor/tor-service-defaults-torrc
`
```
DataDirectory /var/lib/tor
PidFile /run/tor/tor.pid
RunAsDaemon 1
User debian-tor

ControlSocket /run/tor/control GroupWritable RelaxDirModeCheck
ControlSocketsGroupWritable 1
SocksPort unix:/run/tor/socks WorldWritable
SocksPort 9050

CookieAuthentication 1
CookieAuthFileGroupReadable 1
CookieAuthFile /run/tor/control.authcookie

Log notice syslog

```

Users who have multiple tor-instances configured by `tor-service-defaults-torrc-instances` need
`RunAsDaemon 0`

therefore I did not put it in the seednode torrc.